src/LibraryViewExtension/web/layoutSpecs.json

```diff
@@ -1150,6 +1150,30 @@
               "path": "DSCore.String.Join"
             }
           ],
+          "childElements": [ ]
+        }
+      ]
+    },
+    {
+      "text": "Modifiers",
+      "iconUrl": "",
+      "elementType": "category",
+      "include": [],
+      "childElements": [
+        {
+          "text": "Geometry Color",
+          "iconUrl": "",
+          "elementType": "group",
+          "include": [
+            {
+              "path": "Modifiers.GeometryColor.ByGeometryColor",
```

Comment:
For the migration, that has been done in https://github.com/DynamoDS/Dynamo/pull/7759.
As for the namespace, can we ignore it for now until there are issues that can not be ignored?